sdk/search/Azure.Search/src/Generated/Models/IndexBatch.cs

```diff
@@ -13,6 +13,6 @@ namespace Azure.Search.Models
     public partial class IndexBatch
     {
         /// <summary> The actions in the batch. </summary>
-        public ICollection<IndexAction> Actions { get; set; } = new List<IndexAction>();
+        public IList<IndexAction> Actions { get; set; } = new List<IndexAction>();
```

Comment:
Collection properties must not be settable. Or is that a different PR?

Reply:
I think it's a bit more complicated than that. Some lists are nullable so we have to leave the properties settable but this is something this PR didn't try to address.

Reply:
Hmm, good point. I suppose we can treat an empty collection as null since, an empty JSON array might be expected.